dolphinscheduler-task-plugin/dolphinscheduler-task-spark/src/test/java/org/apache/dolphinscheduler/plugin/task/spark/SparkTaskTest.java

```diff
@@ -41,20 +48,33 @@ public void testBuildCommandWithSparkSql() throws Exception {
         Mockito.when(taskExecutionContext.getExecutePath()).thenReturn("/tmp");
         Mockito.when(taskExecutionContext.getTaskAppId()).thenReturn("5536");
 
-        SparkTask sparkTask = Mockito.spy(new SparkTask(taskExecutionContext));
-        sparkTask.init();
-        Assertions.assertEquals(
-                "${SPARK_HOME}/bin/spark-sql " +
-                        "--master yarn " +
-                        "--deploy-mode client " +
-                        "--conf spark.driver.cores=1 " +
-                        "--conf spark.driver.memory=512M " +
-                        "--conf spark.executor.instances=2 " +
-                        "--conf spark.executor.cores=2 " +
-                        "--conf spark.executor.memory=1G " +
-                        "--name sparksql " +
-                        "-f /tmp/5536_node.sql",
-                sparkTask.getScript());
+        ResourceContext resourceContext = Mockito.mock(ResourceContext.class);
```

Comment:
Overall LGTM, It seems that this ut does not test the case of `spark-submit --master local xxx` or `--master <master-url>`

Reply:
Sure. I will add more case.